Dockerfile

```diff
@@ -7,7 +7,9 @@ FROM ruby:2.6-stretch
 ### docker:dind image much more complicated and didn't lend itself to
 ### also running ruby.
 
-RUN apt-get update -qq && apt-get install -qqy \
+RUN apt-get update -qq && \
+    apt-get upgrade -qqy && \
```

Comment:
We should use `dist-upgrade` instead of `upgrade` because `upgrade` has the potential to leave a package at an older version if upgrading it would mean removing a different (less important) package.

Reply:
TIL